docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1016,7 +991,9 @@ The following table lists all the server state attributes the client can receive
 | `MC_PLAYBACK_STATE_MOVING_TO_PREVIOUS` | Playback state of moving to previous media (Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable) |
 | `MC_PLAYBACK_STATE_FAST_FORWARDING` | Playback state of fast forwarding (Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable) |
 | `MC_PLAYBACK_STATE_REWINDING`       | Playback state of rewinding (Tizen 4.0 and Higher for Mobile and Tizen 5.0 and Higher for Wearable) |
-
+| `MC_PLAYBACK_STATE_CONNECTING`  | Connecting (Tizen 6.0 and Higher)         |
+| `MC_PLAYBACK_STATE_BUFFERING`   | Buffering (Tizen 6.0 and Higher)          |
+| `MC_PLAYBACK_STATE_ERROR`       | Error  (Tizen 6.0 and Higher)             |
```

Comment:
@nawab-samsung Thanks for the suggestion. However in this format, I worry it can give an impression that some attributes are allowed to specific Tizen version only. 
For example a user can be confused to think `MC_PLAYBACK_STATE_NONE`  is not allowed in  Tizen 6.0 and higher. 

Also, while separating the tables, the table captions get too long.

So I don't think separating the table into 3 tables is a good idea.

Reply:
@mijin85cho thanks for your suggestion.